agilex_piper/scene.xml

```diff
@@ -0,0 +1,21 @@
+<mujoco model="piper_scene">
+  <include file="piper.xml" />
+  <option integrator="implicitfast" impratio="1" />
```

Comment:
Can you put `option` inside piper.xml? Also impratio="1" is the default so remove it.